src/components/SpecialResolution/Resolution.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <v-card
+    v-if="showSpecialResolutionResolution"
+    id="resolution"
+    flat
+  >
+    <!-- Header -->
+    <article class="header-container section-container">
+      <v-icon color="appDkBlue">
+        mdi-handshake
+      </v-icon>
+      <label class="font-weight-bold pl-2">Special Resolution</label>
+    </article>
+
+    <v-card
+      flat
+      :class="{'invalid-section': invalidResolutionSection}"
+    >
+      <InstructionalText />
+
+      <HelpResolution />
+
+      <section
+        id="resolution-section"
+        class="section-container"
+      >
+        <div
+          id="resolution-header"
+        >
+          <v-row
+            no-gutters
+            class="mt-4"
+          >
+            <v-col>
+              <h2>Special Resolution</h2>
+              <v-chip
+                v-if="isCoopCorrectionFiling && hasChanged"
```

Comment:
Would it be better to move `isCoopCorrectionFiling` to the v-row itself? See also lines 52 and 65.

Reply:
We need it to hide for the base filing, but show up for the corrections, so we need the row.